services/vision/mlvision/ml_model.go

```diff
@@ -220,26 +254,7 @@ func getLabelsFromMetadata(md mlmodel.MLMetadata) []string {
 		if labelPath == "" { // no label file specified
 			return nil
 		}
-		var labels []string
-		f, err := os.Open(filepath.Clean(labelPath))
-		if err != nil {
-			return nil
-		}
-		defer func() {
-			if err := f.Close(); err != nil {
-				logger := logging.NewLogger("labelFile")
-				logger.Warnw("could not get labels from file", "error", err)
-				return
-			}
-		}()
-		scanner := bufio.NewScanner(f)
-		for scanner.Scan() {
-			label := strings.TrimSpace(scanner.Text())
-			if label == "" {
-				continue
-			}
-			labels = append(labels, scanner.Text())
-		}
+		labels := getLabelsFromFile(labelPath)
```

Comment:
Okay cool so this'll be overwritten by anything explicitly put in the Extras of metadata... making it non-breaking. I like this.